.gitlab-ci.yml

```
@@ -0,0 +1,24 @@
+stages:
+  - generator
+  - runTests
+
+generateIntegrationTests:
+  stage: generator
+  image: "julia:1.9"
```

Should probably use 1.10?
```suggestion
  image: "julia:1.10"
```

---

Actual yes, but I don't test Julia 1.10 yet. I want to add Julia 1.10 support in an extra PR. And there, I will also increase the Julia versions for jobs like this.

---

:+1